.github/workflows/combine-prs.yml

```diff
@@ -0,0 +1,150 @@
+# Adapted from https://github.com/hrvey/combine-prs-workflow
+name: 'Combine PRs'
+
+# Controls when the action will run - in this case triggered manually
+on:
+  workflow_dispatch:
+    inputs:
+      branchPrefix:
+        description: 'Branch prefix to find combinable PRs based on'
+        required: true
+        default: 'dependabot'
+      combineBranchName:
```

Comment:
question - so we want to have a PR which has every dependabot update, regardless of whether is passing tests (I see we have omitted `mustBeGreen`)? I guess we will still have the existing dependabot PRs so we can see where to focus our search for the combined PR???

Reply:
We don't have mustBeGreen as a variable, but what we've removed from the code is the path where mustBeGreen was set to false. Now it's hard-coded that the branch must be green to be included in the amalgamation, there's no way of bypassing it. The reason being that it was really hard to have a boolean variable that defaulted to true when the workflow was called by the scheduler (in which case this `inputs` section is ignored), and it seemed to me unlikely we'd ever want to combine failing PRs.